runners/katacoda/index.ts

```diff
@@ -45,6 +45,9 @@ export class Katacoda extends Runner {
         this.setupDir = path.join(this.tempPathTutorial, "setup");
         this.createFolder(this.setupDir, false);
 
+        //set working direktory
+        this.setVariable("workingDir", path.join("/root"));
+
```

Comment:
For the strings "workingDir" and "useDevonCommand" you can create contants and use them in setVariable() and getVariable(). Then it is easier to change the name later and it protects against spelling mistakes.